tensilelite/Tensile/Components/LocalRead.py

```diff
@@ -72,12 +72,10 @@ def __call__(self, writer, kernel, bufferIdx, iui, epsi, tP):
         # for TLU=0 case, blockWidth and LRVW should match
         numReadsPerUnroll = ceil(tP["bpe"] * kernel["MIInputPerThread%s"%tc] / int(unrollBlockWidth * 4)) # bytes/register
         numVgpr  = int(ceil(blockWidth))
+        numElementPerRead = int(blockWidth * 4) // tP['bpe'] if not writer.states.archCaps["HasEccHalf"] else 1
```

Comment:
Why is it related with HasEccHalf?
Could you just fix the UnrollStride in line 66?
With "UnrollStride = int(unrollBlockWidth * 4) // tP["bpe"]"